src/bin/server.ts

```diff
@@ -101,6 +101,8 @@ interface Args {
     }
   }
 
+  options["emitValueUpdateAfterSetValue"] = true;
```

Comment:
I think it'd be better to only set it if the option was not already set, or only default it if a config file was not specified. This takes away the ability for a host application to leave it disabled it if that's the preference. Up to this point, an application could assume that the defaults were aligned with the driver defaults.

It would probably be useful to start documenting any options that differ from the driver default.